.github/issue_template.md

```diff
@@ -0,0 +1,40 @@
+Note: This is just a template, so feel free to use/remove the unnecessary things
+
+### Description
+- Type: Bug | Enhancement\Feature Request | Question
+- Priority: Blocker | Major | Minor
+
+---------------------------------------------------------------
+## Bug
+
+**OS**  
+linux|windows|??
```

Comment:
@sbutcher-arm OK, but shouldn't the mbed-OS related issues be raised in the mbed-OS repository?